src/graphql/hooks.generated.ts

```diff
@@ -2683,8 +2683,17 @@ export const ShippingZoneFragmentDoc = gql`
   }
   name
   description
+  priceRange {
+    start {
+      ...Money
+    }
+    stop {
+      ...Money
+    }
+  }
 }
```

Comment:
question: do you still use it?

Reply:
No, but removing it would require removing whole logic regarding `priceRange` column, and there was a decision to leave it until filtering is implemented on this view